docs/sql_statements/utility_statements.md

```diff
@@ -0,0 +1,48 @@
+Table of Contents
+=================
+
+   * [DESCRIBE Statement](#describe-statement)
+
+# DESCRIBE Statement
+The DESCRIBE and EXPLAIN statements are synonyms, used either to obtain information about table structure or query execution plans. For more information, see [EXPLAIN Statement](#explain-statement).
+
+# EXPLAIN Statement
+`Syntax`
+```
+{EXPLAIN | DESCRIBE | DESC}
+    tbl_name [col_name | wild]
```

Comment:
```
explainable_stmt: {
    SELECT statement
  | DELETE statement
  | INSERT statement
  | REPLACE statement
  | UPDATE statement
}
```

Reply:
The complete syntax will be added on next `explain` patch, here we first support sql `desc table ...`